backend/auth.py

```diff
@@ -38,10 +40,9 @@ def register():
             redis_client.incr('next_uid')
             redis_client.hmset("user{}".format(next_uid), {'fname': fname, 'lname': lname, 'email': email, 'password': generate_password_hash(password), 'isTutor': isTutor, 'uid': next_uid})
             redis_client.bgsave()
-            return redirect(url_for('auth.login'))
-
-        flash(error)
+            return json.dumps({'error': False}), 200, {'Content-Type':'application/json'}
```

Comment:
for this I've been using flasks response method:
https://tedboy.github.io/flask/generated/generated/flask.Response.html
May want to use this to keep everything uniform.